homework7/Animal.java

```diff
@@ -0,0 +1,58 @@
+package org.hillel.homework7;
+
+public abstract class Animal {

```

Comment:
It's absolutely ok to use abstract class for that, I agree. 
Could you please write another solution based on inheritance only (not using abstract class) and compare both solutions? Would be good if you'll write down some notes via your insights and comparison ideas.